### PR TITLE
[0.71] Add a couple of mac specific exports to the TS exports

### PR DESCRIPTION
**Pull request by acoates-ms**
When upgrading FURN to 0.71 I hit a couple of missing exports on the TypeScript type definitions for 0.71

### CODE REVIEW

File: Libraries/Components/AccessibilityInfo/AccessibilityInfo.d.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ type AccessibilityChangeEventName =
   | 'change' // deprecated, maps to screenReaderChanged
   | 'boldTextChanged' // iOS-only Event
   | 'grayscaleChanged' // iOS-only Event
+  | 'highContrastChanged' // [MacOS]
```
Comment (Saadnajmi) [inline Libraries/Components/AccessibilityInfo/AccessibilityInfo.d.ts:18]: Nit: lowercase M

**Review by Saadnajmi — APPROVED** [review]

File: Libraries/Components/TextInput/TextInput.d.ts [hunk 2]
```diff
@@ -274,6 +274,113 @@ export interface TextInputIOSProps {
   scrollEnabled?: boolean | undefined;
 }
 
+// [MacOS
```
Comment (harrieshin) [inline Libraries/Components/TextInput/TextInput.d.ts:277]: yup same comment with Saad on lower case m for "macOS

File: Libraries/Types/CoreEventTypes.d.ts [hunk 3]
```diff
@@ -261,3 +261,23 @@ export interface PointerEvents {
   onPointerUp?: ((event: PointerEvent) => void) | undefined;
   onPointerUpCapture?: ((event: PointerEvent) => void) | undefined;
 }
+
+export interface NativeKeyEvent {
```
Comment (harrieshin) [inline Libraries/Types/CoreEventTypes.d.ts:265]: this ain't macOS only to require // [macOS] ?

**Review by harrieshin — APPROVED** [review]

File: Libraries/Types/CoreEventTypes.d.ts (hunk 3 above)
Comment (Saadnajmi) [reply Libraries/Types/CoreEventTypes.d.ts:265]: It is, and also needs the diff tag.

**Pull request merged**